recipes/torch_em/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: torch_em
+  version: {{ version }}
+
+source:
+  url: https://github.com/constantinpape/torch_em/archive/{{ version }}.tar.gz
+  sha256: "0ec220518c96177e36e52eaad5491b12daecf82e229eade2c1ffd8ea6d0bf711"
+
+build:
+  number: 0
+  script: "pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: true  # [win]
```

Since this is pure Python, better to make this noarch and we can ignore the Windows faillure:
```suggestion
  noarch: python
```